### PR TITLE
Skeleton implementation with hardcoded states

### PR DESCRIPTION
**Pull request by retroinspect**
Very rough implementation of Page 0 UI

### CODE REVIEW

**Comment by retroinspect** [comment]
Rebased to resolve conflict @ikicha

File: web/page0/src/app/safe-device-url.pipe.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import {Pipe, PipeTransform} from '@angular/core';
+import {DomSanitizer} from '@angular/platform-browser';
+
+@Pipe({
+  name: 'safedeviceurl',
+})
+export class SafeDeviceUrlPipe implements PipeTransform {
```
Comment (ikicha) [inline web/page0/src/app/safe-device-url.pipe.ts:7]: is it necessary?

File: web/page0/src/app/runtime-card/runtime-card.component.html [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+        </button>
+    </div>
+
+    <div class="runtime-card-footer">
+        <button mat-icon-button matTooltip="Unregister runtime">
+            <span class="google-symbols"> delete </span>
+        </button>
+    </div>
+
+    <mat-card-content *ngIf="runtime?.status === 'valid'">
```
Comment (ikicha) [inline web/page0/src/app/runtime-card/runtime-card.component.html:24]: is it for placeholder?

File: web/page0/src/app/safe-device-url.pipe.ts (hunk 1 above)
Comment (retroinspect) [reply web/page0/src/app/safe-device-url.pipe.ts:7]: No. I'll remove

**Review by ikicha — APPROVED** [review]

**Pull request merged**